openedx/core/djangolib/translation_utils.py

```diff
@@ -8,13 +8,21 @@ def translate_date(date, language, date_format='DATE_FORMAT'):
     its value for the given language.  Both the format of the date
     as well as its values (i.e., name of the Month) are translated.
 
+    If language is Spainish, This is used to work around a bug in
+    the Spanish django month translations.
+    See EDUCATOR-2328 for more details.
```

Comment:
Nit: I think this should say "If language is Spanish, then the entire date string is returned in lowercase. This is used to work around a bug in..."

Reply:
Got it, will update the PR. 
Who can be a 2nd reviewer?